modules/market/src/main/java/com/opengamma/strata/market/interpolator/MathBoundCurveInterpolator.java

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright (C) 2015 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.market.interpolator;
+
+import com.opengamma.strata.collect.ArgChecker;
+import com.opengamma.strata.collect.array.DoubleArray;
+import com.opengamma.strata.math.impl.interpolation.Interpolator1D;
+import com.opengamma.strata.math.impl.interpolation.data.Interpolator1DDataBundle;
+
+/**
+ * The standard implementation of a bound interpolator.
+ */
+final class MathBoundCurveInterpolator
```

Comment:
I'm not sure about the class name. What does `Math` imply in this context? All interpolators use maths.